gulp-tasks/reference-docs/get-latest-tags.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
```

Comment:
Please add a JSDoc description to this file, include at least `@fileoverview` and `@author`

Reply:
Done.